src/cli.ts

```diff
@@ -6,8 +6,13 @@ const P = new Pokedex();
 
 const fetchPokemonInfo = async (name: string) => {
   console.log(chalk.green(`Fetching information about ${name}...`));
-  const pokemonSpecies = await P.getPokemonSpeciesByName(name);
-  return pokemonSpecies;
+  try {
+    const pokemonSpecies = await P.getPokemonSpeciesByName(name);
```

Comment:
We can actually use the `.toLowerCase()` function on `name` and then we can remove the `console.log()` about ensuring a Pokemon name is all lowercase. A user could then provide `"Charmander"` and `.toLowerCase()` changes it to `"charmander"`.